evap/results/templates/results_course_detail.html

```diff
@@ -35,16 +35,28 @@ <h3 class="mr-auto">{{ course.name }} ({{ course.semester.name }})</h3>
                     title="{% trans 'Shows all results available for you.' %}">
                     {% trans 'Myself' %}
                     </a>
-                    {% if not sufficient_votes_warning and course.is_private %}
-                        <a href="{% url 'results:course_detail' course.semester.id course.id %}?public_view=true" role="button" class="btn btn-sm btn-light{% if public_view %} active{% endif %}" data-toggle="tooltip" data-placement="top"
-                            title="{% trans 'Shows results available for the participants.' %}">
-                            {% trans 'Participant' %}
-                        </a>
-                    {% elif not sufficient_votes_warning %}
-                        <a href="{% url 'results:course_detail' course.semester.id course.id %}?public_view=true" role="button" class="btn btn-sm btn-light{% if public_view %} active{% endif %}" data-toggle="tooltip" data-placement="top"
-                            title="{% trans 'Shows results available for everyone logged in.' %}">
-                            {% trans 'Public' %}
-                        </a>
+                    {% if sufficient_votes_warning %}
+                        <span class="d-inline-block" tabindex="0" data-toggle="tooltip" data-placement="top" title="{% blocktrans %}The results of this course have not been published because it didn't get enough votes.{% endblocktrans %}">
+                            <a href="#" role="button" class="btn btn-sm btn-light disabled" style="pointer-events: none;">
+                                {% if course.is_private %}
+                                    {% trans 'Participant' %}
+                                {% else %}
+                                    {% trans 'Public' %}
+                                {% endif %}
+                            </a>
+                        </span>
+                    {% else %}
+                        {% if course.is_private %}
+                            <a href="{% url 'results:course_detail' course.semester.id course.id %}?public_view=true" role="button" class="btn btn-sm btn-light{% if public_view %} active{% endif %}" data-toggle="tooltip" data-placement="top"
+                                title="{% trans 'Shows results available for the participants.' %}">
+                                {% trans 'Participant' %}
+                            </a>
+                        {% else %}
+                            <a href="{% url 'results:course_detail' course.semester.id course.id %}?public_view=true" role="button" class="btn btn-sm btn-light{% if public_view %} active{% endif %}" data-toggle="tooltip" data-placement="top"
```

Comment:
Isn't this exactly the same line as in line 50? Could you move it out of the if block? Same for the `</a>`